events/templates/event.html

```diff
@@ -42,17 +43,44 @@ <h6 class="float-right inline"> posted by:
     <!-- Users Signed Up -->
     {% with signed_up=event.signed_up.all %}
       <p>Number signed up: {{ signed_up.count }}</p>
-      <table class="table table-hover">
-        {% for atendee in signed_up %}
-          <tr>
-            <td>
-              <a href="{% url 'profile' atendee.id %}">
-                {{atendee.username}}
-              </a>
-            </td>
-          </tr>
-        {% endfor %}
-      </table>
+
+        <!-- Button trigger modal -->
+        <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#exampleModalLong">
+          Users Attending
+        </button>
+
+        <!-- Modal -->
+        <div class="overflow-auto">
+          <div class="modal fade" id="exampleModalLong" tabindex="-1" role="dialog" aria-labelledby="exampleModalLongTitle" aria-hidden="true">
+            <div class="modal-dialog" role="document">
+              <div class="modal-content">
+
+                <div class="modal-header">
+                  <h5 class="modal-title" id="exampleModalLongTitle"> Users List </h5>
+                  <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                  </button>
+                </div>
+
+                <div class="modal-body">
+
+                  {% for atendee in signed_up %}
+                        <a href="{% url 'profile' atendee.id %}">
+                          <img src="{{ atendee.profile.image.url }}" class="rounded-circle" height="40px" alt="">
+                          <span class="ml-1">{{atendee.username}}</span></a>
+                          <br>
+                  {% endfor %}
+
+                </div>
```

Comment:
outer div does not need overflow-auto. We want the list to be scrollable.
'modal-body' needs overflow-auto class and attribute, style='max-height:50vh'
vh is vertical height. Right now it is not bounded so it will never overflow.

Reply:
If you want to test the scrolling feature, go to the /admin panel and edit an event. Add all existing users to that event. In your HTML, temporarily add a bunch of <br> tags to space out the few users we have.